test/mongoConnectionStrings.test.ts

```diff
@@ -5,55 +5,80 @@
 
 // The module 'assert' provides assertion methods from node
 import * as assert from 'assert';
-import { getDatabaseNameFromConnectionString } from '../src/mongo/mongoConnectionStrings';
+import { getDatabaseNameFromConnectionString, addDatabaseToConnectionString } from '../src/mongo/mongoConnectionStrings';
```

Comment:
> I think that my tests changes may be useful for future development in case if any other part of the connection string will be needed

Not a fan of "planning for the future" for tests. I'd rather keep it simple and only test the existing stuff. We can make it more complicated once it's actually necessary